db/seeds.rb

```diff
@@ -6,10 +6,16 @@
              admin: true,
              birthday: '1994-08-21')
 
-10.times do
+2.times do
   User.create!(email: Faker::Internet.email,
                password: Faker::Internet.password(8),
                is_male: Faker::Boolean.boolean,
                admin: false,
                birthday: Faker::Date.birthday(18, 65))
 end
+
+7.times do
+  Interest.create!(name: Faker::ProgrammingLanguage.name,
+                   type: 'Work',
+                   user_id: Faker::Number.between(1,3))
```

Comment:
It's a bad practice to assume what id a record will have.
Think of that in that way: what if
- someone will populate his DB with seeds,
- then he will delete all users from his DB,
- and then he will try to use seeds again?

Ids of the newly created users (those from running seeds the second time) will be 3 and 4, and as so creating interests will fail, as it will try to assign numbers between 1 and 3 for theirs user_ids.
It would be much better to use ids of the real users from the DB here.
And performance in seeds is not important at all, so you can use the simplest approach available (e.g. `User.all.shuffle.first`).